PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalRequest.kt

```diff
@@ -94,6 +98,9 @@ abstract class PayPalRequest internal constructor(
     open var enablePayPalAppSwitch: Boolean = false,
     open var userAuthenticationEmail: String? = null,
     open var userPhoneNumber: PayPalPhoneNumber? = null,
+    open var recurringBillingDetails: PayPalRecurringBillingDetails? = null,
+    open var recurringBillingPlanType: PayPalRecurringBillingPlanType? = null,
+    open var amountBreakdown: RecurringBillingAmountBreakdown? = null,
```

Comment:
This should only be on the `PayPalCheckoutRequest` not the `PayPalRequest` since it is not supported for the `PayPalVaultRequest`